### PR TITLE
Upgrade instance class to F2 #6501

### LINKED ISSUE #6501: Upgrade instance class to F2
To improve performance, we can upgrade the [instance class](https://cloud.google.com/appengine/docs/about-the-standard-environment#instance_classes) to F2. However, I'm unable to do that using the GAE admin console any more http://stackoverflow.com/questions/29295703/why-i-cant-see-instance-class-on-application-settings

I think we need to upgrade our config files. I suspect our files follow a legacy format that existed when we started.

### PR DESCRIPTION
**Pull request by whipermr5**
Fixes #6501

Add instance-class option in appengine config template

### CODE REVIEW

File: src/main/webapp/WEB-INF/appengine-web.template.xml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <appengine-web-app xmlns="http://appengine.google.com/ns/1.0">
 
-  <threadsafe>true</threadsafe>
-    
+    <threadsafe>true</threadsafe>
+    <instance-class>F2</instance-class>
```
Comment (damithc) [inline src/main/webapp/WEB-INF/appengine-web.template.xml]: Let's make it `F1` in this file as devs don't need to use F2 instances. I'll change to F2 in the production version.

**Comment by boxin-yang** [comment]
in https://cloud.google.com/appengine/docs/java/config/appref#scaling_elements_automatic_scaling , it seems that F1 is the default option for GAE(automatic-scaling is assumed by default and automatic scaling has a default instance of F1)

**Comment by whipermr5** [comment]
I am aware. Should I instead leave F2 but comment out the line and explain that it should be uncommented to activate F2?

**Comment by damithc** [comment]
> Should I instead leave F2 but comment out the line and explain that it should be uncommented to activate F2?

I guess no harm leaving the line as it is now. There's no change in behavior right? Making it explicit means easier to change if needed.

**Comment by whipermr5** [comment]
Yup no change in behaviour.

**Comment by wkurniawan07** [comment]
@boxin-yang thanks for chipping in!

**Pull request merged**